src/app/app.js

```diff
@@ -1,12 +1,35 @@
-import React from 'react';
-
-function App() {
-  return (
-    <div>
-      Hello world!
-    </div>
-  );
+import React, {Component} from 'react';
+import { connect } from 'react-redux';
+
+import { getPhotos } from '../actions/index.js';
+import Image from '../components/image.js';
+
+const mapStateToProps = state => ({
+  photos: state.photos.list,
+});
+
+const mapDispatchToProps = {
+  getPhotos,
+};
+
+class App extends Component {
+  componentDidMount() {
+    this.props.getPhotos();
+  }
+  render() {
+    return (
+          <div className="container">
+            <h3 className="center-align">Photo Map App</h3>
+            <Image photos={this.props.photos}/>
+          </div>
+    );
+  }
 }
 
+App.propTypes = {
+  getPhotos: React.PropTypes.func,
+  photos: React.PropTypes.array,
+};
 
-export default App;
+export default connect(mapStateToProps, mapDispatchToProps)(App);
```

Comment:
Another personal preference but I like keep the mapStateToProps mapDispatchToProps function and object near the connect function their used in.